x-pack/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/components/detail_panel/detail_panel.js

```diff
@@ -335,9 +335,7 @@ export class DetailPanelUi extends Component {
                   <EuiButton
                     fill
                     color="primary"
-                    onClick={() => {
-                      routing.navigate(encodeURI(`/auto_follow_patterns/edit/${encodeURIComponent(autoFollowPattern.name)}`));
-                    }}
+                    href={routing.getAutoFollowPatternPath(autoFollowPattern.name)}
```

Comment:
Would it be better to use `routing.getRouterLinkProps(routing.getAutoFollowPatternPath(autoFollowPattern.name))` here so we apply both `href` and `onClick` props to this button?

Reply:
unfortunately `getRouterLinkProps` and `getAutoFollowPatternPath` overlap with each other in terms of concatenating base path, so it breaks when used in conjunction. there's not much benefit to having `onClick` in this case, anyway!